requirements.txt

```diff
@@ -1,249 +1,18 @@
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.5/en_core_web_sm-2.2.5.tar.gz
```

Comment:
Did you make sure this new `requirements.txt` actually works first? Anytime you change this file you need to check that it actually works, as `pip install -r requirements.txt` is known to frequently break due to random package conflicts. To do so, create a temporary conda environment with minimal installs (only python and pip) and run `pip install -r requirements.txt`. If it works, good. Then try running some of your source scripts in this environment to ensure they work as well, and don't have weird issues with package versions coming up. As I said, anytime you change `requirements.txt` you should do this, because dependencies are very important to get right, and very finicky.

Reply:
Checked new `requirements.txt` with new conda environment. Ran `pip install -r requirements.txt` and there are no issues with the install. I ran tests (with new conda environment) on scripts in `src/data` and `src/visualization` and they worked.